### PR TITLE
[ADD] Module to send out timesheet notifications and subscribe the employee's manager

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

File: hr_timesheet_notifications/model/hr_timesheet_sheet.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+        # Subscribe the manager to comments and sheet confirmation
+        data_obj = self.pool['ir.model.data']
+        subtype_ids = [
+            data_obj.get_object_reference(cr, uid, 'mail', 'mt_comment')[1],
+            data_obj.get_object_reference(
+                cr, uid, 'hr_timesheet_notifications', 'subtype_confirm')[1],
+            ]
+
+        if sheet.employee_id.parent_id.user_id:
```
Comment (pedrobaeza) [inline hr_timesheet_notifications/model/hr_timesheet_sheet.py]: Has an employee always a supervisor? On contrary, this will fail.

**Comment by pedrobaeza** [comment]
A little question, but the rest :+1:

**Comment by StefanRijnhart** [comment]
You're absolutely right, I need to check parent_id first as it is not a required field. Thanks!

**Comment by pedrobaeza** [comment]
As a note, with v8, this is not needed, because it propagates False (or [] for one2many and many2many) through empty recordset, avoiding these long expressions.

**Comment by mdietrichc2c** [comment]
@StefanRijnhart is this PR still in WIP state?
Also, it would be good to rebase it on the main 7.0 branch.

**Comment by StefanRijnhart** [comment]
Rebased and fixed bug signalled by @pedrobaeza. Thank you for your patience!

**Comment by pedrobaeza** [comment]
Thanks again :+1:

**Comment by max3903** [comment]
:+1:

**Pull request merged**